developer-docs-site/docs/concepts/coin-and-token/nft-minting-tool.md

```diff
@@ -52,7 +52,7 @@ Now `nft_mint` is your user profile name.
 
 The most popular tool for generating NFT assets is `HashLips`. You can generate NFT assets with `HashLips` or another tool of your choosing. The `aptos-mint` command works with the images and metadata generated by `HashLips` out of the box.
 
-To make it easy to follow along, we uploaded sample assets to [hashlips_assets.zip](https://storage.googleapis.com/nft-mint/hashlips_assets.zip).
+To make it easy to follow along, we uploaded sample assets to [hashlips_assets.zip](https://github.com/aptos-labs/token/releases/download/sample-hashlips-assetss/hashlips_assets.zip).
```

Comment:
I am pointing out this directory name appears misspelled yet the link seems to work: `assetss`

Reply:
Yeah, @jjleng let's create a new release tag without the typo, so people aren't confused